src/patternfly/components/Sequence/sequence.scss

```diff
@@ -0,0 +1,344 @@
+  display: flex;
+  justify-content: center;
+
+  // remove scrollbar
+  ::-webkit-scrollbar {
+    width: 0;
+    height: 0;
+    background: transparent;
+  }
+}
+
+.pf-c-sequence__control {
+  top: 0;
+  bottom: 0;
+  padding-top: var(--pf-c-sequence__control--PaddingTop);
+  padding-bottom: var(--pf-c-sequence__control--PaddingBottom);
+
+  &::before {
+    content: "";
+    border-top: var(--pf-c-sequence__control--BorderTopWidth) solid var(--pf-c-sequence__control--BorderTopColor);
+    border-bottom: var(--pf-c-sequence__control--BorderBottomWidth) solid var(--pf-c-sequence__control--BorderBottomColor);
+  }
+
+  &.pf-m-hover,
+  &:hover {
+    border-top: var(--pf-c-sequence__control--hover--BorderTopWidth) solid var(--pf-c-sequence__control--hover--BorderTopColor);
+    border-bottom: var(--pf-c-sequence__control--hover--BorderBottomWidth) solid var(--pf-c-sequence__control--hover--BorderBottomColor);
+  }
+
+  &.pf-m-active,
+  &:active {
+    border-top: var(--pf-c-sequence__control--active--BorderTopWidth) solid var(--pf-c-sequence__control--active--BorderTopColor);
+    border-bottom: var(--pf-c-sequence__control--active--BorderBottomWidth) solid var(--pf-c-sequence__control--active--BorderBottomColor);
+  }
+
+  &.pf-m-focus,
+  &:focus {
+    border-top: var(--pf-c-sequence__control--focus--BorderTopWidth) solid var(--pf-c-sequence__control--focus--BorderTopColor);
+    border-bottom: var(--pf-c-sequence__control--focus--BorderBottomWidth) solid var(--pf-c-sequence__control--focus--BorderBottomColor);
+  }
+
+  button {
+    padding: 0;
+    background-color: transparent;
+    border: none;
+
+    &:focus {
+      outline-style: none;
+    }
+  }
+}
+
+.pf-c-sequence__control-m-start {
+  left: var(--pf-c-sequence__control-m-start--PositionLeft);
+  order: 1;
+  border-right: var(--pf-c-sequence__control-m-start--BorderRightWidth) solid var(--pf-c-sequence__control-m-start--BorderRightColor);
+}
+
+.pf-c-sequence__control-m-end {
+  right: var(--pf-c-sequence__control-m-end--PositionRight);
+  order: 3;
+  border-left: var(--pf-c-sequence__control-m-end--BorderLeftWidth) solid var(--pf-c-sequence__control-m-end--BorderLeftColor);
+}
+
+.pf-c-sequence__control-icon {
+  min-width: var(--pf-c-sequence__control-icon--MinWidth);
+  color: var(--pf-c-sequence__control-icon--Color);
+}
+
+// Horizontal list
+.pf-c-sequence__list {
+  display: inline-flex;
+  order: 2;
+  overflow: scroll;
+}
+
+// Change default styles for the list and list items
+.pf-c-sequence__list .pf-c-sequence__item {
+  list-style-type: none;
+}
+
+// Items inside of list
+.pf-c-sequence__item {
+  max-width: var(--pf-c-sequence__item--MaxWidth);
+
+  &:not(:last-child) {
+    margin-right: var(--pf-c-sequence__item--MarginRight);
+  }
+}
+
+// Sequence Link Position Number
+.pf-c-sequence__number {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  min-width: var(--pf-c-sequence__number--MinWidth);
```

Comment:
I've noticed that with a number with two digits the circle grows. In the design it shows "10" as still in a circle. With this solution the circle will become an oval. This could be brought back to design if we don't have a good solution. I know we investigated the psuedo element concept a bit but I'm not sure where that landed.

Reply:
I did see the weird styling with the 10. What Im thinking right now is if we change min-width and max-height to width and height, then there is no room for the circle to change to an oval shape, when the content gets too wide. What do you think @matthewcarleton

Reply:
ya this is a tricky one to solve, if the font size grows then it breaks. Which isn't good for accessibility.

Reply:
@matthewcarleton is it possible to make the width and height % of the font size.